MobiFlight/Board.cs

```diff
@@ -279,6 +322,36 @@ public class Board
         /// </summary>
         public UsbDriveSettings UsbDriveSettings;
 
+        /// <summary>
+        /// Base path for custom firmware
+        /// </summary>
+        public String BasePath;
+
+        /// <summary>
+        /// The image resource
+        /// </summary>
+        public Image BoardImage;
```

Comment:
You'll need to have two separate properties here when you merge in the JSON validation. One should be `BoardImagePath` and the other should be `BoardImage`, which is marked to not JSON serialize.

Reply:
Wait, never mind, I see that it's a fixed file name. This will just have to get marked to not serialize when you merge in the JSON schema validation.